charts/civil-ccd/values.preview.template.yaml

```diff
@@ -57,6 +57,7 @@ civil-service:
       OIDC_ISSUER: https://forgerock-am.service.core-compute-idam-aat2.internal:8443/openam/oauth2/realms/root/realms/hmcts
       SERVICE_REQUEST_UPDATE: https://${SERVICE_NAME}.preview.platform.hmcts.net/service-request-update
       SERVICE_REQUEST_UPDATE_CLAIM_ISSUED: https://${SERVICE_NAME}.preview.platform.hmcts.net/service-request-update-claim-issued
+      ROLE_ASSIGNMENT_URL: http://am-role-assignment-service-aat.service.core-compute-aat.internal
```

Comment:
Should this not have a variable in to swap out per env ?

Reply:
I can see it's already in values.template.yaml, so not sure we need it here...

Reply:
I've moved it from civil-service in values.preview.template.yaml to civil-service in values.template.yaml